### PR TITLE
Update PROTOCOL-WEB.md

### PR DESCRIPTION
**Pull request by wenbozhu**
Clarify the content type requirement (for message formats).

### CODE REVIEW

File: doc/PROTOCOL-WEB.md [hunk 1]
```diff
@@ -37,6 +37,7 @@ Content-Type
 
 1. application/grpc-web
   * e.g. application/grpc-web+[proto, json, thrift]
+  * the message format, e.g. +proto, +json, should always be specified by the sender
```
Comment (anuraaga) [inline doc/PROTOCOL-WEB.md]: How about also adding something like this?
```
* for misbehaving clients that don't specify the format, the server will always default to proto
```

**Comment by wenbozhu** [comment]
@anuraaga  PTAL

File: doc/PROTOCOL-WEB.md [hunk 2]
```diff
@@ -37,6 +37,8 @@ Content-Type
 
 1. application/grpc-web
   * e.g. application/grpc-web+[proto, json, thrift]
+  * the sender should always specifiy the message format, e.g. +proto, +json
```
Comment (fengli79) [inline doc/PROTOCOL-WEB.md]: specify

File: doc/PROTOCOL-WEB.md [hunk 3]
```diff
@@ -37,6 +37,8 @@ Content-Type
...
+  * the receiver should assume the default is "+proto" when the message format is mssing in the C-T, i.e. "application/grpc-web"
```
Comment (anuraaga) [inline doc/PROTOCOL-WEB.md]: missing

Also seems worth writing out Content-Type

**Comment by wenbozhu** [comment]
@fengli79  @anuraaga PTAL

**Comment by anuraaga** [comment]
LGTM

**Pull request merged**